src/composables/layer-metadata/layer-metadata.composable.ts

```diff
@@ -0,0 +1,162 @@
+import { useTranslation } from 'i18next-vue'
+import { ThemeNodeModel } from '../themes/themes.model'
+import { LayerTreeNodeModel } from '@/components/layer-tree/layer-tree.model'
+import { IdValues, LayerMetadataModel } from './layer-metadata.model'
+import {
+  getMetadataLinks,
+  getResponsibleParty,
+  isoLang2To3,
+  stringToHtml,
+} from './layer-metadata.utils'
+import useWmsHelper from '../common/wms-helper.composable'
+import useThemes from '../themes/themes.composable'
+
+export default function useLayerMetadata() {
+  // TODO: get urls from a config
+  const geonetworkBaseUrl = 'https://geocatalogue.geoportail.lu/geonetwork/srv'
```

Comment:
we should try to gather all the settings things to be sure not to forget anyone with hard coded values

Reply:
yes, I've created a ticket for that